addOns/spider/src/main/java/org/zaproxy/addon/spider/automation/SpiderJob.java

```diff
@@ -538,7 +546,21 @@ public String getHandleParameters() {
         }
 
         public void setHandleParameters(String handleParameters) {
-            this.handleParameters = handleParameters;
+            this.handleParameters = getHandleParamsOptionEnum(handleParameters).name();
+        }
+
+        private HandleParametersOption getHandleParamsOptionEnum(String handleParametersVisited) {
+            HandleParametersOption option =
+                    EnumUtils.getEnumIgnoreCase(
+                            HandleParametersOption.class, handleParametersVisited);
+            if (option == null) {
+                LOGGER.warn(
+                        "\"{}\" is not a valid HandleParametersOption value, defaulting to \"{}\"",
+                        handleParametersVisited,
+                        HANDLE_PARAMETERS_OPTION_DEFAULT);
+                option = HANDLE_PARAMETERS_OPTION_DEFAULT;
+            }
+            return option;
```

Comment:
These are old changes.

Reply:
UGH, I musta pushed from a wrong copy of the branch. I'll get this fixed/re-submitted.

Reply:
I'm so sorry I totally forgot that I'd worked on this at work. So my home system had the branch in the wrong state when I rebased. I'll open another PR fixing things 🤦‍♂️

Reply:
https://github.com/zaproxy/zap-extensions/pull/5292